tests/integration/test_data_model_replicator.py

```diff
@@ -42,6 +43,12 @@ def test_data_modeling_replicator():
         os.remove("states.json")
     except FileNotFoundError:
         pass
+    delete_state_store_in_cdf(
```

Comment:
Could this use `state_store.delete_state()`?

https://github.com/cognitedata/python-extractor-utils/blob/76f713d944b6a6346142186c478a7e5f3f61eb58/cognite/extractorutils/statestore.py#L243

Reply:
Yes, I added lines to use that instead.  The catch is that the `delete_state` method isn't enough on its own to remove the state from remote storage, you also have to run `synchronize` so that it syncs the remote store.